### PR TITLE
Update logging docs

### PR DESCRIPTION
**Pull request by pierwill**
This PR updates the docs to prioritize `systemd` as the logging system for most new installations.

Addresses #2508.

### CODE REVIEW

File: content/influxdb/v1.7/administration/logs.md [hunk 1]
```diff
@@ -32,38 +32,52 @@ influxd 2>$HOME/my_log_file
...
+{{% /tabs %}}
+{{< tab-content-container >}}
+<!--------------------------- BEGIN systemd  ------------------->
+{{% tab-content %}}
+#### systemd
+
+InfluxDB uses the system-configured default for logging (usually `journald`).
+On most Linux systems, the logs will be directed to the `systemd` journal and can be accessed with the command:
+
+```
```
Comment (sanderson) [inline content/influxdb/v1.7/administration/logs.md]: Add the syntax after the tick marks. Either `sh` or `bash`.

~~~
```sh
~~~

File: content/influxdb/v1.7/administration/logs.md [hunk 2]
```diff
@@ -32,38 +32,52 @@ influxd 2>$HOME/my_log_file
...
+[systemd](#)
+[sysinit](#)
+{{% /tabs %}}
+{{< tab-content-container >}}
+<!--------------------------- BEGIN systemd  ------------------->
+{{% tab-content %}}
+#### systemd
+
+InfluxDB uses the system-configured default for logging (usually `journald`).
+On most Linux systems, the logs will be directed to the `systemd` journal and can be accessed with the command:
```
Comment (sanderson) [inline content/influxdb/v1.7/administration/logs.md]: - Use present tense and active voice.

> Most Linux systems direct logs to the `systemd` journal.
> To access these logs, use the command:

File: content/influxdb/v1.7/administration/logs.md [hunk 3]
```diff
@@ -32,38 +32,52 @@ influxd 2>$HOME/my_log_file
...
+
+```
+sudo journalctl -u influxdb.service
+```
+
+See the manual pages for `systemd` and `journald` for more information about configuring `journald`.
```
Comment (sanderson) [inline content/influxdb/v1.7/administration/logs.md]: - For more information about configuring `journald`, see the...
- Maybe add links to the actual man pages.

File: content/influxdb/v1.7/administration/logs.md [hunk 4]
```diff
@@ -32,38 +32,52 @@ influxd 2>$HOME/my_log_file
...
+{{% /tab-content %}}
+<!--------------------------- END systemd  --------------------->
+<!--------------------------- BEGIN sysvinit  ------------------>
+{{% tab-content %}}
 #### sysvinit
 
-If InfluxDB was installed using a prebuilt package, and then launched
-as a service, `stderr` is redirected to `/var/log/influxdb/influxd.log` and all log data will be written to that file.
+On older Linux sytems, all log data (and `stdedrr`) will be written to `/var/log/influxdb/influxd.log`.
```
Comment (sanderson) [inline content/influxdb/v1.7/administration/logs.md]: - I don't know if I'd use "older" here.
- Use present tense and active voice.
- ~`stdedrr`~ -> `stderr`

> Some Linux systems write log and **stderr** data to `/var/log/influxdb/influxd.log`.

File: content/influxdb/v1.7/administration/logs.md [hunk 5]
```diff
@@ -32,38 +32,52 @@ influxd 2>$HOME/my_log_file
...
 For example, if `/etc/default/influxdb` contains:
 
 ```
 STDERR=/dev/null
 ```
 
-all log data will be discarded.  Likewise, you can direct output to
-`stdout` by setting `STDOUT` in the same file.  Output to `stdout` is
-sent to `/dev/null` by default when InfluxDB is launched as a service.
+all log data will be discarded.
```
Comment (sanderson) [inline content/influxdb/v1.7/administration/logs.md]: all log data ~will be~ **is** discarded.

File: content/influxdb/v1.7/administration/logs.md [hunk 6]
```diff
@@ -32,38 +32,52 @@ influxd 2>$HOME/my_log_file
...
+Likewise, you can direct output to `stdout` by setting `STDOUT` in the same file.
+Output to `stdout` is sent to `/dev/null` by default when InfluxDB is launched as a service.
```
Comment (sanderson) [inline content/influxdb/v1.7/administration/logs.md]: Opinion only, so feel free to ignore. "Output to `stdout`..." feels redundant. What about:

> **Stdout** is sent to `/dev/null` by default...

File: content/influxdb/v1.7/administration/logs.md [hunk 7]
```diff
@@ -32,38 +32,52 @@ influxd 2>$HOME/my_log_file
...
-write files to `/var/log/influxdb` by default, and will now use the
-system configured default for logging (usually `journald`).  On most
-Linux systems, the logs will be directed to the `systemd` journal and can be accessed with the command:
-
-```
-sudo journalctl -u influxdb.service
-```
-
-See the `systemd` `journald` documentation about configuring `journald`.
+>**Note:** macOS logs are stored, by default, in the file `/usr/local/var/log/influxdb.log`.
```
Comment (sanderson) [inline content/influxdb/v1.7/administration/logs.md]: Maybe include a title with this note so it's more easily scannable.

~~~
> ##### macOS log location
> macOS stores logs in `/usr/local/var/log/influxdb.log` by default.
~~~

File: content/influxdb/v1.7/administration/logs.md [hunk 8]
```diff
@@ -32,38 +32,53 @@ influxd 2>$HOME/my_log_file
...
+To access these logs, use this command:
+
+```sh
+sudo journalctl -u influxdb.service
+```
 
-If InfluxDB was installed using a prebuilt package, and then launched
-as a service, `stderr` is redirected to `/var/log/influxdb/influxd.log` and all log data will be written to that file.
-You can override this location by setting the environment variable `STDERR` in the InfluxDB configuration file `/etc/default/influxdb`.
+For more information about configuring `journald`, see its [manual page](https://www.freedesktop.org/software/systemd/man/journald.conf.html).
```
Comment (sanderson) [inline content/influxdb/v1.7/administration/logs.md]: For offsite links, I try to make sure link text clearly communicates where the link is going (so it doesn't feel like an ambiguous advertising link).

> ..., see the [journald manual page](https://www.freedesktop.org/software/systemd/man/journald.conf.html).

File: content/influxdb/v1.7/administration/logs.md [hunk 9]
```diff
@@ -32,38 +32,53 @@ influxd 2>$HOME/my_log_file
...
+{{% /tab-content %}}
+<!--------------------------- END systemd  --------------------->
+<!--------------------------- BEGIN sysvinit  ------------------>
+{{% tab-content %}}
+#### sysvinit
 
->**Note:** macOS logs are stored, by default, in the file `/usr/local/var/log/influxdb.log`.
+On Linux sytems not using systemd, InfluxDB writes all log data and `stderr` to `/var/log/influxdb/influxd.log`.
+You can override this location by setting the environment variable `STDERR` in the InfluxDB configuration file (`/etc/default/influxdb`).
```
Comment (sanderson) [inline content/influxdb/v1.7/administration/logs.md]: - If this is set in the config file, it's not an "environment" variable. Environment variables are set in the environment. This is just a setting or variable in the config file. I wasn't aware this was a thing – setting variables in the config file. I don't see anywhere where `STDERR` is set or reference in the sample config file.
- Clarify that the _default_ location of the InfluxDB config file is `/etc/default/influxdb.conf` (I believe we include the `.conf` extension in all distros).

File: content/influxdb/v1.7/administration/logs.md [hunk 10]
```diff
@@ -32,38 +32,54 @@ influxd 2>$HOME/my_log_file
...
+For more information, see the [`journald.conf` manual page](https://www.freedesktop.org/software/systemd/man/journald.conf.html).
+{{% /tab-content %}}
+<!--------------------------- END systemd  --------------------->
+<!--------------------------- BEGIN sysvinit  ------------------>
+{{% tab-content %}}
+#### sysvinit
 
->**Note:** macOS logs are stored, by default, in the file `/usr/local/var/log/influxdb.log`.
+On Linux sytems not using systemd, InfluxDB writes all log data and `stderr` to `/var/log/influxdb/influxd.log`.
+You can override this location by setting the environment variable `STDERR` in `/etc/default/influxdb`.
```
Comment (sanderson) [inline content/influxdb/v1.7/administration/logs.md]: Ook, so `/etc/default/influxdb` is just a file that InfluxDB reads when starting up? If so, I think we need to call out this behavior. If this is how it works, it makes sense to me now.

File: content/influxdb/v1.7/administration/logs.md [hunk 11]
```diff
@@ -32,38 +32,54 @@ influxd 2>$HOME/my_log_file
...
-
-See the `systemd` `journald` documentation about configuring `journald`.
+{{% /tab-content %}}
+<!--------------------------- END sysvinit --------------------->
+{{< /tab-content-container >}}
+{{< /tab-labels >}}
+<!--------------------------- END TABS  ------------------------>
+
+> #### macOS log location
+> macOS logs are stored by default at `/usr/local/var/log/influxdb.log`.
```
Comment (sanderson) [inline content/influxdb/v1.7/administration/logs.md]: This is really nit-picky, sorry. "macOS logs" aren't stored here. These are where InfluxDB logs are stored when running on macOS.

**Comment by pierwill** [comment]
(Closes #2508)

**Pull request merged**